README.rst

```diff
@@ -824,15 +824,19 @@ The configuration of this feature is stored in the configuration variable *trace
 .. code-block:: python
 
     traceability_checklist = {
-        'private_token': config('PRIVATE_TOKEN'),
-        'api_host_name': config('API_HOST_NAME'),
-        'project_id': config('PROJECT_ID'),
-        'merge_request_id': config('MERGE_REQUEST_ID'),
-        'attribute_name': config('TARGET_ATTRIBUTE_NAME'),
-        'attribute_to_str': config('TARGET_ATTRIBUTE_TO_STRING'),
-        'attribute_values': config('TARGET_ATTRIBUTE_VALUES'),
+        'private_token': config('PRIVATE_TOKEN', ''),
+        'api_host_name': config('API_HOST_NAME', 'https://api.github.com'),
```

Comment:
Lets not put config here, but rather hardcoded strings and then a comment behind them what should be in the field (or somehow else). Make sure to mention that if they are empty then this or that feature is disabled, but warnings should still be there if we use any checklist directives..

Reply:
To be done in issue https://github.com/melexis/sphinx-traceability-extension/issues/144